pylink/jlink.py

```diff
@@ -968,14 +965,12 @@ def enable_dialog_boxes(self):
           ``None``
         """
         self.exec_command('SetBatchMode = 0')
+        self.exec_command("HideDeviceSelection = 0")
```

Comment:
I think you'll also need to re-enable the info windows: `EnableInfoWinFlashBPs`, and `EnableInfoWinFlashDL`.